ranger/container/directory.py

```diff
@@ -252,6 +252,19 @@ def hidden_filter_func(fobj):
                         return False
                 return True
             filters.append(hidden_filter_func)
+        if self.settings.global_inode_type_filter and not self.inode_type_filter:
+            # pylint: disable=too-many-boolean-expressions
+            def hidden_inode_filter_func(obj):
+                g_filter = self.settings.global_inode_type_filter
+                if (
+                        ('d' in g_filter and obj.is_directory) or
```

Comment:
Could we either use use a few if-else branches with one `return` in each or return a single expression? Right now it's a strange hybrid.

```python
if foo:
    return True
elif bar:
    return True
else:
    return False
```
or
```python
return foo or bar
```

Not:
```python
if foo or bar:
    return True
else:   # or even no else
    return False
```

I'd slightly prefer the first version.